causalpy/pymc_experiments.py

```diff
@@ -336,15 +340,301 @@ def plot(self, counterfactual_label="Counterfactual", round_to=None, **kwargs):
+        cumulative_results = self.post_y.sum()
+        _mu_samples_cumulative = (
+            self.post_pred["posterior_predictive"]
+            .mu.stack(sample=("chain", "draw"))
+            .sum("obs_ind")
+        )
+
+        # Mean calculations
+        mean_results = self.post_y.mean()
+        _mu_samples_mean = (
+            self.post_pred["posterior_predictive"]
+            .mu.stack(sample=("chain", "draw"))
+            .mean("obs_ind")
+        )
+
+        # Posterior Mean
+        results["posterior_estimation"] = {
+            "cumulative": round(np.mean(_mu_samples_cumulative.values), 2),
+            "mean": round(np.mean(_mu_samples_mean.values), 2),
+        }
+
+        results["results"] = {
+            "cumulative": round(cumulative_results, 2),
+            "mean": round(mean_results, 2),
+        }
+
+        results["_systematic_differences"] = {
+            "cumulative": results["results"]["cumulative"]
+            - results["posterior_estimation"]["cumulative"],
+            "mean": results["results"]["mean"]
+            - results["posterior_estimation"]["mean"],
+        }
+
+        if correction:
+            _mu_samples_cumulative += results["_systematic_differences"]["cumulative"]
+            _mu_samples_mean += results["_systematic_differences"]["mean"]
+
+        results["ci"] = {
+            "cumulative": [
+                round(np.percentile(_mu_samples_cumulative, ci), 2),
+                round(np.percentile(_mu_samples_cumulative, 100 - ci), 2),
+            ],
+            "mean": [
+                round(np.percentile(_mu_samples_mean, ci), 2),
+                round(np.percentile(_mu_samples_mean, 100 - ci), 2),
+            ],
+        }
+
+        cumulative_upper_mde = round(
+            results["ci"]["cumulative"][1]
+            - results["posterior_estimation"]["cumulative"],
+            2,
+        )
+        cumulative_lower_mde = round(
+            results["posterior_estimation"]["cumulative"]
+            - results["ci"]["cumulative"][0],
+            2,
+        )
+
+        mean_upper_mde = round(
+            results["ci"]["mean"][1] - results["posterior_estimation"]["mean"], 2
+        )
+        mean_lower_mde = round(
+            results["posterior_estimation"]["mean"] - results["ci"]["mean"][0], 2
+        )
+
+        results["posterior_mde"] = {
+            "cumulative": round((cumulative_upper_mde + cumulative_lower_mde) / 2, 2),
+            "mean": round((mean_upper_mde + mean_lower_mde) / 2, 2),
+        }
+        return results
+
+    def power_summary(
+        self, alpha: float = 0.05, correction: bool = False
+    ) -> pd.DataFrame:
+        """
+        Summarize the power estimation results in a DataFrame format.
+
+        This function calls '_power_estimation' to perform power estimation calculations and then
+        converts the resulting dictionary into a pandas DataFrame for easier analysis and visualization.
+
+        Parameters:
```

Comment:
Same comment as above about the parameters and returns blocks not rendering properly in when you build and check the docs locally.